x-pack/platform/plugins/shared/cases/server/tasks/incremental_id/incremental_id_task_manager.ts

```diff
@@ -0,0 +1,143 @@
+              );
+              // Increment the case ids
+              const processedAmount = await this.casesIncrementService.incrementCaseIds(
+                casesWithoutIncrementalId
+              );
+              this.logger.debug(
+                `Applied incremental ids to ${processedAmount} out of ${casesWithoutIncrementalId.length} cases`
+              );
+
+              const endTime = performance.now();
+              this.logger.debug(
+                `Task terminated ${CASES_INCREMENTAL_ID_SYNC_TASK_ID}. Task run took ${
+                  endTime - startTime
+                }ms [ started: ${initializedTime}, ended: ${new Date().toISOString()} ]`
+              );
+            },
+            cancel: async () => {
+              this.casesIncrementService?.stopService();
+              this.logger.debug(`${CASES_INCREMENTAL_ID_SYNC_TASK_ID} task run was canceled`);
+            },
+          };
+        },
+      },
+    });
+  }
+
+  public setupIncrementIdTask(taskManager: TaskManagerStartContract, core: CoreStart) {
+    this.taskManager = taskManager;
+    try {
+      // Instantiate saved objects client
+      const internalSavedObjectsRepository = core.savedObjects.createInternalRepository([
+        CASE_SAVED_OBJECT,
+        CASE_ID_INCREMENTER_SAVED_OBJECT,
+      ]);
+      const internalSavedObjectsClient = new SavedObjectsClient(internalSavedObjectsRepository);
+
+      this.casesIncrementService = new CasesIncrementalIdService(
+        internalSavedObjectsClient,
+        this.logger
+      );
+
+      this.taskManager
+        .ensureScheduled({
+          id: CASES_INCREMENTAL_ID_SYNC_TASK_ID,
+          taskType: CASES_INCREMENTAL_ID_SYNC_TASK_TYPE,
+          schedule: {
+            interval: CASES_INCREMENTAL_ID_SYNC_INTERVAL_DEFAULT,
+          },
+          params: {},
+          state: {},
+          scope: ['cases'],
+        })
+        .then(
+          (taskInstance) => {
+            this.logger.info(
+              `${CASES_INCREMENTAL_ID_SYNC_TASK_ID} scheduled with interval ${taskInstance.schedule?.interval}`
+            );
+          },
+          (e) => {
+            this.logger.error(
+              `Error scheduling task: ${CASES_INCREMENTAL_ID_SYNC_TASK_ID}: ${e}`,
+              e?.message ?? e
+            );
+          }
+        );
+    } catch (e) {
+      this.logger.error(
+        `Error running task: ${CASES_INCREMENTAL_ID_SYNC_TASK_ID}: ${e}`,
+        e?.message ?? e
+      );
+      return null;
+    }
+  }
+
+  /**
+   * Ensure the id incrementer task is running soon
+   */
+  public async scheduleIdCrementerTask() {
+    try {
+      const taskInstance = await this.taskManager?.get(CASES_INCREMENTAL_ID_SYNC_TASK_ID);
+      if (taskInstance?.status === TaskStatus.Idle) {
+        await this.taskManager?.runSoon(CASES_INCREMENTAL_ID_SYNC_TASK_ID);
+      }
+    } catch (e) {
+      this.logger.debug(`Could not run task: ${e}`);
```

Comment:
Can we update this to
```suggestion
      this.logger.debug(`Could not run case incremental id task: ${e}`);
```